### ISSUE TITLE
Friendly Name for Windows "Open With" Context Menu

### ISSUE THREAD

**Issue by RoyBeer** [mifi/lossless-cut#1014]
When I tried opening a file via right click, "Open With", I realized the name stretches the context menu into an unsightly mess.

![image](https://user-images.githubusercontent.com/16647738/152535879-f45554ce-dc99-4f61-83e9-53a822d0a690.png)

I simply added a .reg file with the following contents to workaround until then, may someone benefit from it:

```
Windows Registry Editor Version 5.00

[HKEY_CLASSES_ROOT\Applications\LosslessCut-win.exe]
"FriendlyAppName"="LosslessCut"
```

**Comment by mifi** [comment]
Hi. Where did you download the app from? here on github or app store or somehwere else?
I cannot find out how to set "FriendlyAppName" with electron-builder

related: #23 https://github.com/mifi/lossless-cut/commit/f9e43a3bb4d84976bdc61ebdda6a40e19abba5db

**Comment by RoyBeer** [comment]
> Hi. Where did you download the app from? here on github or app store or somehwere else?

Hey, I downloaded the latest release (3.42.0) from this project's release page.

> I cannot find out how to set "FriendlyAppName" with electron-builder
I don't know anything about electron-builder, but from a simple lookup, in which of your files the term `the swiss army knife of lossless video/audio editing` shows up, it might be the `/package.json` - more specific the `description` value.

I don't want to presume to suggest a change of your program's choice of description, but maybe it's as simple as that.
Or you might want to suggest to the user to change it in a fashion like I did.

> related: https://github.com/mifi/lossless-cut/issues/23 https://github.com/mifi/lossless-cut/commit/f9e43a3bb4d84976bdc61ebdda6a40e19abba5db

Okay, maybe next time I should read the related issue before I write an answer.
So yeah, it seems like you just put that description value so it's not empty. With an almost educated guess I would say the "Description" inside that file basically is what Windows then uses as a "FriendlyAppName" inside it's Registry.

Long story short:  
`"description": "The swiss army knife of lossless video/audio editing"` should be something like ` "description": "LosslessCut"`

**Comment by mifi** [comment]
Yea I thought about changing that, but I was also worried that it's being used some other place. I know it will be used in npm, but because it's not published in npm, that's not an issue.

I think this is rather a bug in electron-builder, because why would it choose the `description` field as the `Open With` association.

Need to dig a big more and maybe file an issue at electron-builder. Worst case we can set `description` to LosslessCut

**Comment by shodanx2** [comment]
Hello,

I have a feature to suggest.
When selecting multiple video files that losslesscut supports (mp4, other ext?)

As secondary context menu option in explorer

"Merge files with losslesscut"

For single files 
I would like "Cut with Losslesscut"

![image](https://github.com/mifi/lossless-cut/assets/10621885/bfe2eb39-38ef-49da-b6c0-aaca679137f7)


For multiple files I would like "Cut with Losslesscut" and open each file in its own instance
"Merge files with losslesscut" where all the files are open to merge in a single instance

**Comment by mifi** [comment]
is this still a problem as of the newest version?
